tasks/main.yml

```diff
@@ -64,6 +66,17 @@
 
 # Create cores, if any are configured.
 - include_tasks: cores.yml
-  when: "solr_cores and solr_version.split('.')[0] >= '5'"
+  when:
+    - solr_cores
+    - solr_version.split('.')[0] >= '5'
+    - solr_jetty_host == '127.0.0.1'
+
+# Create cores also when solr is listening on custom network interface. Just for solr 9.3.0 and higher.
+- include_tasks: cores-post93.yml
```

Comment:
What's the requirement for a separate cores task file here? I don't think that was explicitly explained but it includes a bit of duplicate code.